website/docs/r/cluster.html.markdown

```diff
@@ -362,6 +362,7 @@ The following attributes are exported:
 * `canal_network_provider` - (Optional) Canal network provider config (list maxitems:1)
 * `flannel_network_provider` - (Optional) Flannel network provider config (list maxitems:1)
 * `weave_network_provider` - (Optional) Weave network provider config (list maxitems:1)
+* `mtu` - (Optional) Network provider MTU. Default `0` (int)
```

Comment:
Is it zero or 1500?

Reply:
mtu `0` lets cni mtu value untouched. It's got from host.